deb/debian-buster/Dockerfile

```diff
@@ -1,14 +1,15 @@
 ARG GO_IMAGE
 ARG DISTRO=debian
 ARG SUITE=buster
-ARG BUILD_IMAGE=${DISTRO}:${SUITE}
+ARG BUILD_IMAGE=${DISTRO}:${SUITE}-backports
 
 FROM ${GO_IMAGE} AS golang
 
 FROM ${BUILD_IMAGE}
 
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y curl devscripts equivs git
+ARG SUITE
+RUN apt-get update && apt-get install -y curl devscripts equivs git debhelper/${SUITE}-backports dwz/${SUITE}-backports
```

Comment:
what's the `dwz/` ? Is that something defined by `deb_helper` ?

Reply:
Ah yeah, I should've explained better -- `dwz` is a dependency used by `debhelper`, and when we pull in the newer `debhelper`, we also need the newer `dwz` (the failure noted in https://github.com/docker/docker-ce-packaging/pull/686#issuecomment-1121001857 is related -- it needs the same change, but Raspbian has no backports so uh, this gets complicated and might force this PR to be reworked / abandoned for now :see_no_evil:).

Reply:
We could go to `debhelper-compat` version 12 instead, and then Buster would no longer need backports, but then we run into bugs with `dh_dwz` (which frankly, even if it works correctly, won't be doing anything), so we'd have to also add `override_dh_dwz` that's a no-op.  It's probably worth updating this to that so you can see that (perhaps in a separate commit so you can see both?).